lib/pbench/agent/modules/metaclass.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+import json
+import os
+
+
+class BadConstruction(Exception):
```

Comment:
All you need in Python for sub-classing `Exception` is:

```
class BadConstruction(Exception):
    pass
```

The constructor and string conversion is handled by default.

Reply:
Can we use the name for the class as something like, `ToolMetadataExc`?